src/richie/apps/courses/models/organization.py

```diff
@@ -131,7 +131,10 @@ class OrganizationPluginModel(PagePluginMixin, CMSPlugin):
         Page,
         on_delete=models.CASCADE,
         related_name="organization_plugins",
-        limit_choices_to={"publisher_is_draft": True, "organization__isnull": False},
+        limit_choices_to={
+            "publisher_is_draft": True,  # plugins work with draft instances
```

Comment:
I think we should add the condition:
```python
"node__parent__cms_pages__publisher_is_draft": True, # organization has published parent
```

Reply:
To enable the use of sub-organization 👍

Reply:
Adding the condition breaks the test... i'm manually testing to understand why.

Reply:
So no need for this condition. Even if the organization is a child organization, they'll appear in the list. The 2 existing conditions are sufficient.

Reply:
no the purpose of this is not show organizations that are below a page that is not published... as it should be considered unpublished if its parent is hidden.

Reply:
Adding` "node__parent__cms_pages__publisher_is_draft": True ` breaks existing tests and does not appear to be useful functionally. Let's keep the conditions for organization as is for now.